src/Compilers/CSharp/Test/Semantic/Semantics/DelegateTypeTests.cs

```diff
@@ -14011,6 +14025,10 @@ public void ParamsArray_SynthesizedDelegateIL()
                 			int32[] arg2
                 		) runtime managed 
                 	{
+                		.param [2]
+                			.custom instance void [{{s_libPrefix}}]System.ParamArrayAttribute::.ctor() = (
```

Comment:
>.custom instance void [{{s_libPrefix}}]System.ParamArrayAttribute::.ctor() = ( [](http://example.com/codeflow?start=19&length=78)

Are we testing this for method groups rather than for lambdas?

Reply:
The emitted delegate should be exactly the same, no matter what the source is (lambda or method). I added a test to show that: 019a75bda0659b6c987dea662054d4ba03400cbf